frontend/src/model/beta/Releases.tsx

```diff
@@ -1,18 +1,32 @@
 import { Box, Button, Stack } from '@mui/material'
 import { useEffect, useState } from 'react'
 
+import { useGetReleasesForModelId } from '../../../actions/release'
+import { ModelInterface, ReleaseInterface } from '../../../types/types'
+import EmptyBlob from '../../common/EmptyBlob'
+import Loading from '../../common/Loading'
 import DraftNewReleaseDialog from '../DraftNewReleaseDialog'
 import ModelReleaseDisplay from '../ModelReleaseDisplay'
 
-export default function Releases({ model }: { model: any }) {
+export default function Releases({ model }: { model: ModelInterface }) {
   const [latestRelease, setLatestRelease] = useState<string>('')
   const [openDraftNewRelease, setOpenDraftNewRelease] = useState(false)
 
+  const { releases, isReleasesLoading } = useGetReleasesForModelId(model.id)
+
+  const sortByReleaseVersionDescending = (a: ReleaseInterface, b: ReleaseInterface) => {
```

Comment:
This sort of thing is nice to have available as a reusable piece of logic. Consider pulling out into a sorting utils file.